app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -727,14 +727,16 @@ export default {
             case 'custom':
               if (obj.component.validate.custom.length) {
                 validatorIdentity +=
-                  '|custom:' + obj.component.validate[validity].trim();
+                  '|custom:' +
+                  obj.component.validate[validity].trim().replaceAll(',', ';');
```

Comment:
This is going to make the data in the report different from what was submitted, which might confuse people and cause them to submit bug reports to us. What about putting `validatorIdentity` and the `error_message` into double-quotes instead?